src/components/Main.jsx

```diff
@@ -0,0 +1,198 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaw, faCirclePlus } from "@fortawesome/free-solid-svg-icons";
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import TodoItem from "./TodoItem";
+
+const Main = () => {
+  const initialData = localStorage.getItem("data")
+    ? JSON.parse(localStorage.getItem("data"))
+    : [];
```

Comment:
```suggestion
  const initialData = JSON.parse(localStorage.getItem("data") || []);
```

```suggestion
  const initialData = JSON.parse(localStorage.getItem("data") ?? []);
```
이렇게 두 방법으로 더 간단하게 나타낼 수 있겠네요~